lib/Manifestation.inc

```diff
@@ -59,6 +57,16 @@ class Manifestation extends ManifestationMaster {
     return (is_array($result)) ? reset($result) : $result;
   }
 
+  /**
+   * @return string|null
+   */
+  public function getTitleExtra() {
+    $specific = $this->getTitleSpecific();
+    $full = $this->getTitle();
+
+    return ($specific !== $full) ? $full : NULL;
```

Comment:
Så hvis de to er identiske, så får man ingenting?

Det kræver en kommentar i koden :-)

Reply:
Jep - så skal den ikke vises (ellers kommer der bare til at stå det samme to gange), jeg smider en kommentar ind :)